crates/revm/src/db/states/state_builder.rs

```diff
@@ -18,8 +18,8 @@ pub struct StateBuilder<'a, DBError> {
     /// This will initialize cache to this state.
     pub with_cache_prestate: Option<CacheState>,
     /// Do we want to create reverts and update bundle state.
-    /// Default is true.
-    pub without_bundle_update: bool,
+    /// Default is false.
+    pub with_bundle_update: bool,
     /// Do we want to merge transitions in background.
     /// This will allows evm to continue executing.
     /// Default is false.
```

Comment:
Should all of these be private(pub crate) instead?
I don't think they should be mut accessible from the outside

Reply:
Makes sense